app/controllers/items_controller.rb

```diff
@@ -1,4 +1,5 @@
 class ItemsController < ApplicationController
+  before_filter :set_item, only: [:show, :edit, :update, :destroy]
```

Comment:
Your Gemfile is using Rails 4.2.5.1, so it's YAGNI to see Rails 5 right now, unless you want to upgrade.